model/context.go

```diff
@@ -13,114 +13,219 @@ import (
 // function, all the other applicable fields in the Context can
 // inferred and populated from the id of any one of the fields.
 type Context struct {
-	Task       *task.Task
-	Build      *build.Build
-	Version    *version.Version
-	Patch      *patch.Patch
-	ProjectRef *ProjectRef
+	projectRef *ProjectRef
```

Comment:
Line 60 (and possibly others) of base_angular.html also need to be changed since you changed the casing of the context variables